lib/sequelize.js

```diff
@@ -145,6 +149,7 @@ var Sequelize = function(database, username, password, options) {
     pool: {},
     quoteIdentifiers: true,
     hooks: {},
+    retry: {max: 5, match: ['SQLITE_BUSY: database is locked']},
```

Comment:
I'm just tryng to consider if there's any possible issues with that, but nothing springs to mind.